### ISSUE TITLE
Spurious `Could not find module`

### ISSUE THREAD

**Issue by etorreborre** [atom-haskell/haskell-ghc-mod#22]
Open [this project](https://github.com/ambiata/p). And modify `P.hs` on line 37 for example to introduce an error
```
                   , nulle
```

I get 9 errors from that
```
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 12, 18
Could not find module ‘P.Foldable’
Use -v to see a list of the files searched for.
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 17, 18
Could not find module ‘P.Function’
Use -v to see a list of the files searched for.
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 16, 18
Could not find module ‘P.List’
Perhaps you meant List (needs flag -package haskell98-2.0.0.3)
Use -v to see a list of the files searched for.
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 15, 18
Could not find module ‘P.Ord’
Use -v to see a list of the files searched for.
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 14, 18
Could not find module ‘P.Monoid’
Use -v to see a list of the files searched for.
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 13, 18
Could not find module ‘P.Maybe’
Perhaps you meant Maybe (needs flag -package haskell98-2.0.0.3)
Use -v to see a list of the files searched for.
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 11, 18
Could not find module ‘P.Either’
Use -v to see a list of the files searched for.
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 10, 18
Could not find module ‘P.Bool’
Use -v to see a list of the files searched for.
/Users/etorreborre/projects/ambiata/p/private/Users/etorreborre/projects/ambiata/p/src/P.hs: 9, 18
Could not find module ‘P.Applicative’
Use -v to see a list of the files searched for.
```

If I press `CTRL+s` again I get
```
/Users/etorreborre/projects/ambiata/p/src/P.hs: 37, 22
Module ‘Data.List’ does not export ‘nulle’
```

as expected. If I fix the bug and save again, I get 9 errors (the same as before) and saving one more time removes all errors.

**Comment by lierdakil** [comment]
Please also specify what versions of GHC and GHC-Mod you're running. Thanks.

**Comment by lierdakil** [comment]
I'm asking because I can't reproduce on ghc-mod version 5.2.1.2 and GHC 7.8.4.

**Comment by etorreborre** [comment]
I'm using GHC 7.8.4 and ghc-mod fbe0800 (master about 2 weeks ago). I can't use ghc-mod 5.2.1.2 because it doesn't work with GHC 7.8.4 and Cabal 1.22.4.0 (which I need because of the way Setup.hs works for now).

**Comment by lierdakil** [comment]
Oh. That explains much. I should teach myself to ask this question first.

I published ide-haskell v0.5.9, which should help with this particular problem, but please see #18.

**Comment by etorreborre** [comment]
Things seems to be working completely fine with this new version, thanks!
